Makefile

```diff
@@ -0,0 +1,60 @@
+# simple makefile to simplify repetitive build env management tasks under posix
+
+# caution: testing won't work on windows, see README
+
+PYTHON ?= python
+PYTESTS ?= pytest
+
+all: clean inplace test
+
+clean-pyc:
+	find . -name "*.pyc" | xargs rm -f
+
+clean-so:
+	find . -name "*.so" | xargs rm -f
+	find . -name "*.pyd" | xargs rm -f
```

Comment:
is this needed? what does it do?

Let's keep this Makefile minimal ... see also "clean-ctags", "clean-e", etc.

Please remove all commands that you do not need.